wsd/ClientSession.cpp

```diff
@@ -799,7 +799,8 @@ bool ClientSession::_handleInput(const char *buffer, int length)
             }
             else
             {
-                docBroker->updateLastModifyingActivityTime();
+                if (!isReadOnly())
+                    docBroker->updateLastModifyingActivityTime();
```

Comment:
"moveselectedclientparts: Moves the selected parts to a new position."

should be that possible in readonly mode?

Reply:
I expect that would modify the document and there not allowed.